Exercise3/.ipynb_checkpoints/JupyterNB_example-checkpoint.ipynb

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
@manasvita, this file is in a hidden directory, should it be tracked or no?

Reply:
I am not entirely sure what this file is. It is not really being used but I just copied the files from the previous bootcamp. Let me double check it

Reply:
@dhml I removed this checkpoint file. All should be good now